cypress-prioritization-and-triage.md

```diff
@@ -0,0 +1,123 @@
+# Cypress Prioritization and Triage Methodology
+
+At Cypress, we love our open source community. We work every day to grow our partnership with the community through open, honest, and straightforward communication about our processes and plans for the platform. The goal of this document is to provide resources that show our plans for the future and serve as a guide to our processes around handling issues that are raised by the open source community.
+
+## Table of Contents
+
+- [Cypress Priorities](#priorities)
+- [Cypress Roadmap](#cypress-roadmap)
+- [Cypress Issue Triage](#cypress-issue-triage)
+  - [Cypress Triage Process](#cypress-triage-process)
+- [How We Prioritize Issues](#how-we-prioritize-issues)
+- [FAQs](#faqs)
+  - [Why Isn't Anyone Working on My Issue?](#why-isnt-anyone-working-on-my-issue)
+  - [Why Did You Close My Ticket?](#why-did-you-close-my-ticket)
+  - [Backpatching Strategy/Limitations](#backpatching-strategylimitations)
+- [How You Can Help](#how-you-can-help)
+
+
+## Cypress Priorities
+
+Cypress is constantly looking to improve and grow our testing platform to meet the ever-growing needs of the testing world. Like any software platform, we must balance research, development, and maintenance against real-world resource constraints. We feel that transparency about our future plans will help us grow a stronger relationship with end users who understand that all choices involve trade-offs.
+
+## Cypress Roadmap
+
+Our [high level roadmap](https://github.com/orgs/cypress-io/projects/13/views/1) shows high-level projects at Cypress. 
```

Comment:
note: not yet public

Reply:
Yep, I don't plan to merge this until we make that public first

Reply:
```suggestion
Our [Cypress App Priorities](https://github.com/orgs/cypress-io/projects/13/views/1) board shows the high-level roadmap at Cypress. 
```